### PR TITLE
added lists, toast and avatar

### PR DESCRIPTION
**Pull request by rohitrout**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -236,6 +255,7 @@ hr{
     align-items: center;
     border-radius: 50%;
   }
+
   .icon-button.avatar{
       height: 70px;
       width: 70px;
```
Comment (pradipgharge) [inline styles.css]: use rems for length units

File: styles.css [hunk 2]
```diff
@@ -285,11 +305,15 @@ hr{
       border: 1px solid #9b68f1;
       max-width: 300px;
   }
+  .input-val.error{
+    border:1px solid red;
```
Comment (pradipgharge) [inline styles.css]: You can use color variables here

**Review by pradipgharge — APPROVED** [review]
Suggested a few changes, rest looks good to merge

File: styles.css (hunk 1 above)
Comment (rohitrout) [reply styles.css]: Thanks, updated the respective changes

**Pull request merged**